src/components/modules/document/common/Upload/Sides/BackSide/BackSide.vue

```diff
@@ -136,6 +138,8 @@ export default {
     },
 
     handleConfirm(status) {
+      this.$options.riskScoringStore.sendFingerprint();
```

Comment:
Should it be async?

Reply:
non-actual anymore
also not, async not required, because we should not await for it